src/station.cpp

```diff
@@ -717,6 +718,33 @@ Money AirportMaintenanceCost(Owner owner)
 	return total_cost >> 3;
 }
 
+/**
+ * Create or delete the hangar corresponding to this airport.
+ * @param create if true, create a depot associated to this airport;
+ *               if false, delete the hangar corresponding to this airport.
+ */
+void Airport::SetHangar(bool create)
```

Comment:
Why not split this up in two separate functions? An `AddHangar` and `RemoveHangar`. All I see is hardcoded `true` or `false` parameters, so it sounds like there is little need for it to be the same function.